clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -144,7 +145,7 @@ export const TcfOverlay = ({
         const fullExperience: PrivacyExperience = { ...result, ...userPrefs };
 
         setExperience(fullExperience);
-        loadMessagesFromExperience(i18n, fullExperience);
+        loadMessagesFromExperience(i18n, fullExperience, translationOverrides);
```

Comment:
Do we need an equivalent for non-TCF overlay too?

Reply:
no, the minimal -> full experience is TCF only.

Reply:
Or, a better answer would be, the non-TCF experience already does this on initialization (see https://github.com/ethyca/fides/blob/main/clients/fides-js/src/lib/i18n/i18n-utils.ts#L496)

Reply:
ah yes, thank you!